frontend/src/static/js/v_authorship.js

```diff
@@ -41,23 +41,75 @@ window.vAuthorship = {
       totalBlankLineCount: '',
       filesSortType: 'lineOfCode',
       toReverseSortFiles: true,
-      filterSearch: '*',
+      searchBarValue: '',
     };
   },
 
   watch: {
-    filterType() {
-      if (this.filterType === 'checkboxes') {
-        const searchBar = document.getElementById('search');
-        searchBar.value = '';
-        this.filterSearch = '*';
-      } else {
-        this.selectedFileTypes = this.fileTypes.slice();
+    filesSortType() {
+      window.addHash('authorshipSortBy', this.filesSortType);
+      window.encodeHash();
+    },
+
+    toReverseSortFiles() {
+      window.addHash('reverseAuthorshipOrder', this.toReverseSortFiles);
+      window.encodeHash();
+    },
+
+    isLoaded() {
+      if (this.isLoaded) {
+        this.retrieveHashes();
```

Comment:
would it be simpler to call these two methods in `mounted` hook instead?

Reply:
won't work as explained here https://github.com/reposense/RepoSense/pull/1057#issuecomment-604363267 unfortunately

as for the above I moved it to group the 'lifecycle' hash methods together 😮

Reply:
Ahhh I see, sorry for not referring back to the comment.